README.md

```diff
@@ -151,6 +226,60 @@ export default defineConfig({
 });
 ```
 
+### Plugins
+the `@openapi-codegen/cli` supports these generator plugins:
+#### **generateSchemaTypes** (frontend/backend)
+  generate all schema types for your specification:
+  ```ts
+   const { schemasFiles } = await generateSchemaTypes(context, {
+        /* config */
+      });
+  ```
+output: `schemas.ts`
+
+#### **generateFetchers** (frontend)
+  generate all fetchers with types for your specification *needs schemafiles*
+  ```ts
+     await generateFetchers(context, {
+        /* config */
+        schemasFiles,
+      });
+  ```
+output: `fetchers.ts`
+
+#### **generateReactQueryComponents** (frontend)
+  generate all React Query Components for useQuery() and useMutation()
+  ```ts
+      await generateReactQueryComponents(context, {
+        /* config*/
+        schemasFiles,
+      });
+  ```
+  output: `components.ts`
+#### **generateReactQueryFunctions** (frontend)
+  generate all React Query Functions used for e.g. React-Router 6.6.0+ loader functions
+  ```ts
+     await generateReactQueryFunctions(context, {
+        filenamePrefix,
+        schemasFiles,
+      });
+  ```
+  outout: `queryFunctions.ts`
+
+  example useage in react-route-loader:
+  ```ts
+  export const routeLoader = (queryClient: QueryClient) =>
+    async ({ params }: any) => 
+      await queryClient.fetchQuery(...getYourQueryNameQuery({}), {
+        /*options*/
+      })
+  ```
+  *more infos: https://reactrouter.com/en/main/guides/data-libs*
+
+#### **planned plugins**
+- generateExpressControllers (Backend)
+- generateRestfulReactComponents (Frontend)
```

Comment:
Let’s remove this, it’s a side project, "planned plugins" it’s a lie 😅 

Let’s remove them from the example and add the `generateReactQueryFunctions` instead

Reply:
is removed